### ISSUE TITLE
Basic-Chat in iframe does not work on Safari or iOS

### ISSUE THREAD

**Issue by thevalyreangroup** [smilefam/SendBird-JavaScript#71]
Because of the recently added cross-site tracking cookie disabling Apple has done, the use of cookies in the library with sync manager does not work. If you place the served site into an iframe from another domain we get `IDBFactory.open() called in an invalid security context`.

**Comment by chrisallo** [comment]
SyncManager switched store engine from localStorage to IDB (or IndexedDB) due to storage limitation, and IDB access in iframe is genetically forbidden by Safari browser for security concern. It seems like many other libraries which wrap IDB suffer from same problem.

I think there're 2 possible ways to fix it currently:
1) To wrap localStorage as a polyfill when IDB is not available (limitation applies).
2) To discard iframe.

**Comment by thevalyreangroup** [comment]
@chrisallo Right. I figured it might be IDB related but didn't look too deep. I just switched to the non SyncManager version for now. For my context, discarding the iframe was not an option. This is an Angular7 app and i could not get this to integrate with it. So i'm serving the static page from my express backend into an iframe.

**Comment by chrisallo** [comment]
We shall keep this as a known issue for the fix later.
Thanks for your report :)
